examples/llama/requirements.txt

```diff
@@ -1,4 +1,3 @@
 torch
-transformers
 fairscale
 sentencepiece
```

Comment:
Do we also add instructions to install "llama" manually in document?

Reply:
Updated README.md